src/main/java/edu/kit/ipd/crowdcontrol/objectservice/notification/NotificationPolicy.java

```diff
@@ -0,0 +1,34 @@
+package edu.kit.ipd.crowdcontrol.objectservice.notification;
+
+/**
+ * NotificationPolicy encapsulates strategies to check and send notifications.
+ *
+ * @author Simon Korz
+ * @version 1.0
+ */
+public abstract class NotificationPolicy<T> {
```

Comment:
If you mean changing that into a functional interface, one would have to split the interface. From the docs: "Conceptually, a functional interface has exactly one abstract method"